maskwatch-bot/__init__.py

```diff
@@ -95,6 +95,21 @@ async def _is_oper(self, nickname: str):
         })
         return whois_line.command == RPL_WHOISOPERATOR
 
+    async def _get_account(self, nickname: str):
```

Comment:
this is a good idea, but experience tells me that `await`ing a `WHOIS` response for everyone that connects will choke under very high cliconn rate. we're better off firing the `WHOIS` off and handling `RPL_WHOISACCOUNT` in `async def line_read(..)` *if* we get it

Reply:
(declare `User.account` as `account: Optional[str] = None`, then it won't be expected in the constructor)